opensearch-jobs.yml

```diff
@@ -225,6 +299,7 @@ instance_groups:
         - options:
             bucket: (( grab terraform_outputs.logs_opensearch_archive_bucket_name ))
             encoding: gzip
+            codec: 'plain { format => "%{message}" }'
```

Comment:
This seems new? What is this change about?

Reply:
This will fix how we log to S3 it will make it so we no longer are going to need one of the opensearch s3 fixes

Reply:
https://github.com/cloud-gov/opensearch-boshrelease/blob/d053759a4b216ba2e8c8e5dd5397d81afa024323/jobs/ingestor_syslog/templates/config/filters_pre.conf.erb#L32

Reply:
Will this make it more confusing though? How will we know when to apply that filter or when not to? Or will the grok just succeed/fail automatically?

Reply:
actually we should wait on this part, removed for now but once we are doing live i think we should put it back in for future cases